src/test/java/com/pubnub/api/endpoints/objects_api/utils/IncludeTest.java

```diff
@@ -0,0 +1,46 @@
+package com.pubnub.api.endpoints.objects_api.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class IncludeTest {
+    private Include objectUnderTest;
+
+    @BeforeEach
+    void setUp() {
+        objectUnderTest = new Include();
+    }
+
+    @Test
+    void when_inclusionFlags_are_not_empty_then_should_enrich_base_param_map() {
+        objectUnderTest.addInclusionFlag("custom");
+        Map<String, String> baseParams = createBaseParams();
+
+        Map<String, String> enrichedParameterMap = objectUnderTest.enrichParameters(baseParams);
+
+        assertTrue(enrichedParameterMap.containsKey("include"));
+    }
+
+    @Test
+    void when_inclusionFlags_are_empty_then_should_not_enrich_base_param_map() {
+        Map<String, String> baseParams = createBaseParams();
+
+        Map<String, String> enrichedParameterMap = objectUnderTest.enrichParameters(baseParams);
+
+        assertFalse(enrichedParameterMap.containsKey("include"));
```

Comment:
I'd change it in a similar manner as a previous one

Reply:
Yes, I like your improvement.  I will modify it.

Reply:
Is this `assertFalse` necessary?

Reply:
No. I will remove it.